static/app/views/explore/components/schemaHints/schemaHintsDrawer.tsx

```diff
@@ -150,8 +162,20 @@ function SchemaHintsDrawer({hints, searchBarDispatch, queryRef}: SchemaHintsDraw
           onChange={() => handleCheckboxChange(hint)}
         >
           <CheckboxLabelContainer>
-            <Tooltip title={prettifyTagKey(hint.key)} showOnlyOnOverflow skipWrapper>
-              <CheckboxLabel>{prettifyTagKey(hint.key)}</CheckboxLabel>
+            <Tooltip
+              title={
+                hint.kind === FieldKind.FUNCTION
+                  ? `${prettifyTagKey(hint.key)}(...)`
+                  : prettifyTagKey(hint.key)
```

Comment:
this pattern is used in quite a few places, maybe we can pull it out into a function?